<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EventWindow</class>
 <widget class="QDialog" name="EventWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTextEdit" name="description">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>30</y>
     <width>200</width>
     <height>100</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="confirmButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>280</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>確定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>280</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>150</y>
     <width>200</width>
     <height>80</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab1">
    <attribute name="title">
     <string> 指定時間 </string>
    </attribute>
    <widget class="QLineEdit" name="hrText">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>10</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="minText">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>10</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="hrLabel">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>10</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>時</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="minLabel">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>10</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>分</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>60</width>
       <height>30</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>上午</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>下午</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QWidget" name="tab2">
    <attribute name="title">
     <string> 剩餘時間  </string>
    </attribute>
    <widget class="QLineEdit" name="hrText2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="minText2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>10</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="hrLabel2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>10</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>時</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="minLabel2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>10</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>分</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="afterLabel">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>10</y>
       <width>40</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>之後</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="errorLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>240</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
